packages/node_modules/@ciscospark/internal-plugin-wdm/src/device/device.js

```diff
@@ -281,8 +283,9 @@ const Device = SparkPlugin.extend({
       return Promise.reject(new Error('`uri` is a required parameter'));
     }
 
-    for (const value of Object.values(target)) {
-      if (value && uri.startsWith(value)) {
+    for (const key of Object.keys(target)) {
+      const serviceUrl = target[key];
+      if (key !== 'idbroker' && uri.startsWith(serviceUrl)) {
```

Comment:
Same question as above. I think adding a comment as to why `idbroker` is special will clarify.

Reply:
Same reason above.